todo_app/data/trello_utils.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+not_started_board_id = os.getenv("TRELLO_NOT_STARTED_LIST_ID")
+completed_board_id = os.getenv("TRELLO_COMPLETE_LIST_ID")
```

Comment:
(See similar comments above about delaying loading these; making the lists below into a `get_status_for_list_with_id` function or similar might resolve that)